docs/en/setup/backend/ui-setup.md

```diff
@@ -46,4 +31,4 @@ We could set up environment variables to configure this image.
 
 ### SW_OAP_ADDRESS
 
-The address of your OAP server. The default value is `http://127.0.0.1:12800`.
\ No newline at end of file
+The address of your OAP server. The default value is `127.0.0.1:12800`.
```

Comment:
Some ppl would be surprised we don't need `http://` again. Because we just changed from this to `http://xx` when move from zuul to Spring Gateway last year?

Reply:
The things like docs on docker hub, we can't versioning it, https://hub.docker.com/r/apache/skywalking-ui.

Is there any chance we could still keep `http://ip:port` for the Armeria web container?